app/Http/Controllers/CleaningStatusController.php

```diff
@@ -2,9 +2,16 @@
 
 namespace App\Http\Controllers;
 
+use App\Models\Room;
 use Illuminate\Http\Request;
 
-class CleaningStatusController extends Controller
-{
-    //
-}
+class CleaningStatusController extends Controller {
+    public function changeCleaningStatus(Request $request, int $id) {
```

Comment:
graag id naar 'roomId' veranderen, ik weet pas waar het voor is als ik door de functie lees